### PR TITLE
Update static docs site with new theme

### PR DESCRIPTION
**Pull request by mzbeck**
Test plan - Tested locally and with full build environment.

```
== RELEASE NOTES ==

* Changed theme of the Presto docs site (https://prestodb.io/docs/). This adds a table of contents, page navigation bars, Sphinx search, ability to copy code to clipboard, and a new color scheme.
```

### CODE REVIEW

**Comment by caithagoras** [comment]
Presto squash the 2 commits into 1. In Presto, each commit accomplish its own purpose -  we avoid using multiple commits to do one thing.

File: presto-docs/src/main/sphinx/static/versions.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "0.244": "0.244",
+    "0.243.3": "0.243.3",
+    "0.243.2": "0.243.2",
+    "0.243.1": "0.243.1",
+    "0.243": "0.243",
+    "0.242.1": "0.242.1",
+    "0.242": "0.242",
+}
```
Comment (caithagoras) [inline presto-docs/src/main/sphinx/static/versions.json:9]: What is the plan for this file? Are we going to require people to update it for each release?

**Review by caithagoras — APPROVED** [review]

File: presto-docs/src/main/sphinx/static/versions.json (hunk 1 above)
Comment (mzbeck) [reply presto-docs/src/main/sphinx/static/versions.json]: I removed that file; it was unused.

**Comment by caithagoras** [comment]
Please update the commit message. 

Commit title should be less than 72 characters. Commit message guideline: https://chris.beams.io/posts/git-commit/

**Pull request merged**